### PR TITLE
Adding new graffiti types

### PR DESCRIPTION
**Pull request by ZomgPonies**
- Moved the crayon stuff from toys to the crayon file so it's not split
- Added the graffiti found here https://github.com/d3athrow/vgstation13/pull/4936 minus "lambda, chaos, 50 blessings" since they don't fit SS13
- Removed the "cyka" graffiti because it's a russian meme

### CODE REVIEW

**Comment by Nienhaus** [comment]
Those are pretty cool.

**Comment by Keilden** [comment]
Dear gods! Dicks!!!! Please say your not adding those.

**Comment by FalseIncarnate** [comment]
Dicks (on Tg which is where this is from) are only options with brain damage 60+

If we keep them, hopefully we keep the brain damage lock for them too.

:+1: from me for the overall changes/additions

File: code/game/objects/items/crayons.dm [hunk 1]
```diff
@@ -1,3 +1,114 @@
...
+/obj/item/toy/crayon
+	name = "crayon"
+	desc = "A colourful crayon. Looks tasty. Mmmm..."
+	icon = 'icons/obj/crayons.dmi'
+	icon_state = "crayonred"
+	w_class = 1.0
+	attack_verb = list("attacked", "coloured")
+	var/colour = "#FF0000" //RGB
+	var/drawtype = "rune"
+	var/list/graffiti = list("body","amyjon","face","matt","revolution","engie","guy","end","dwarf","uboa","up","down","left","right","heart","borgsrogue","voxpox","shitcurity","catbeast","hieroglyphs1","hieroglyphs2","hieroglyphs3","security","syndicate1","syndicate2","nanotrasen","lie","dick1","dick2","dick3","valid")
```
Comment (Robertha89) [inline code/game/objects/items/crayons.dm]: I would remove the dick ones

**Comment by Fox-McCloud** [comment]
:-1:  until the genitalia is removed.

:+1:  once it is.

**Comment by ZomgPonies** [comment]
How about if I switch it to appear if there's brain damage like in the original PR?

**Comment by Fox-McCloud** [comment]
Don't personally think it's a good idea to encourage players on any level to use that--it's just kinda..asking for trouble, IMO.

**Comment by Robertha89** [comment]
this will end up with the Church of Badmin drawing dicks everywhere ever.

**Comment by TheDZD** [comment]
Yeah, I don't really think we should port over the phallus graffiti. On another note, removing some of the other graffiti because it doesn't fit with SS13 doesn't really make much sense, considering the variety of references already in the game. (Imperium Robes, God Emperor of Mankind genetics disk, etc)

**Comment by Mijhead** [comment]
I agree with DZD. But still, :+1:

**Pull request merged**